tests/integration/seccomp-notify.bats

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	teardown_seccompagent
+	setup_seccompagent
+
+	teardown_busybox
+	setup_busybox
+}
+
+function teardown() {
+	teardown_seccompagent
+	teardown_busybox
+}
+
+@test "runc run (seccomp notify tests)" {
+	requires root
+	requires no_systemd
+	# to fetch the seccomp fd, runc uses the pidfd_getfd system call,
+	# available in Linux >= 5.6
+	# https://github.com/torvalds/linux/commit/8649c322f75c96e7ced2fec201e123b2b073bf09
+	if [[ "$KERNEL_MAJOR" -lt 5 || ("$KERNEL_MAJOR" -eq 5 && "$KERNEL_MINOR" -lt 6) ]]; then
+		skip "requires kernel 5.6"
+	fi
+
+	# The agent intercepts mkdir syscalls and creates the folder appending
+	# "-foo" to the name.
+	update_config '.process.args = ["/bin/sh", "-c", "mkdir /dev/shm/foo && stat /dev/shm/foo-bar"] |
+		.linux.seccomp = {
+			"defaultAction":"SCMP_ACT_ALLOW",
+			"listenerPath": "'$SECCCOMP_AGENT_SOCKET'",
```

Comment:
```suggestion
			"listenerPath": "'"$SECCCOMP_AGENT_SOCKET"'",
```

From Travis:
```
"listenerPath": "'$SECCCOMP_AGENT_SOCKET'",
                  ^--------------------^ SC2086: Double quote to prevent globbing and word splitting.
```

Once the tests pass, I think you can do a draft PR on runc upstream and reference that PR in the runtime-spec PR.

Reply:
Ops, that was my initial idea but I thought so many quotes were confusing. I pushed the new changes, let's wait a bit for travis-ci to pass.